jni/src/test/java/com/horizen/commitmenttreenative/ScCommitmentCertPathTest.java

```diff
@@ -0,0 +1,159 @@
+package com.horizen.commitmenttreenative;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import org.junit.Test;
+
+import com.horizen.librustsidechains.FieldElement;
+
+public class ScCommitmentCertPathTest {
+    private ScCommitmentCertPath getCertPath(CommitmentTree commTree, FieldElement scId, FieldElement certLeaf) {
```

Comment:
whould rename it in something like
 addCertAndReturnPath
for better clarity

Reply:
You are right. I'll fix it

Reply:
Done